### ISSUE TITLE
AccTest: Classic loadbalancer test failure

### ISSUE THREAD

**Issue by wonchulee** [NaverCloudPlatform/terraform-provider-ncloud#296]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform Ncloud Provider Version

```text
Terraform version : v1.5.2
Ncloud Provider version : main(2.3.18)

```

### Affected Resource(s)

* ncloud_load_balancer

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
# None
```

### Debug Output

```
--- FAIL: TestAccNcloudLoadBalancerSSLCertificateBasic (0.14s)
    resource_ncloud_load_balancer_ssl_certificate_test.go:81: Step 1/2 error: Error running pre-apply refresh: exit status 1

        Error: Insufficient rule_list blocks

          on terraform_plugin_test.tf line 9, in resource "ncloud_load_balancer" "lb":
           9: 		resource "ncloud_load_balancer" "lb" {

        At least 1 "rule_list" blocks are required.

        Error: Unsupported argument

          on terraform_plugin_test.tf line 14, in resource "ncloud_load_balancer" "lb":
          14: 			rule_list = [

        An argument named "rule_list" is not expected here. Did you mean to define a
        block of type "rule_list"?
FAIL
FAIL	github.com/terraform-providers/terraform-provider-ncloud/ncloud	0.680s
FAIL
```

### Expected Behavior

Pass the test

### Actual Behavior

Test failure

### Steps to Reproduce

```
export NCLOUD_REGION=KR
export NCLOUD_ACCESS_KEY=xxx
export NCLOUD_SECRET_KEY=xxx
TF_ACC=1 go test ./internal/service/classicloadbalancer/... -run=TestAccNcloudLoadBalancerSSLCertificateBasic -v
```

**Comment by sososo0** [comment]
I'm working on this issue!

**Comment by wonchulee** [comment]
resolved by #321
